timescaledb/load_data.py

```diff
@@ -23,17 +25,19 @@ def create_table(conn):
         CREATE TABLE %s (
             id SERIAL,
             time TIMESTAMPTZ,
-            duration NUMERIC,
             db_version TEXT,
-            branch TEXT,
             query_mode TEXT,
-            build_id TEXT,
+            jenkins_job_id TEXT,
```

Comment:
is the job_id different from the build_id?

Reply:
No. They stand for the same thing.